src/multi-select/multi-select-extended.html

```diff
@@ -0,0 +1,17 @@
+<template>
+  <require from="./styles/multi-select-extended.css"></require>
+
+  <div class="multi-input-container" show.bind="!disabled">
+    <select value.bind="inputValue" change.delegate="addOnChange ? addInput(): null">
+      <option if.bind="placeholder" model.bind="null" selected style="display: none;">${placeholder}</option>
```

Comment:
Deze logica is denk ik niet geschikt voor de multi select. hierdoor verdwijnt de optie met de placeholder eens je iets anders kiest. dit is nuttig voor gewone selects die verplicht zijn, maar niet hier. beter zou zijn om bij het klikken op de plus te kijken of deze optie niet geselecteerd is, alvorens aan de lisjt toe te voegen.

Reply:
ok nvm, de default optie komt terug eens je op de plus geklikt hebt